### PR TITLE
Console api experiments

### PR DESCRIPTION
**Pull request by jacob-statsig**

### CODE REVIEW

**Comment by jasonwzm** [comment]
🚀 Deployed on https://62cf5376c689c95a5a84714b--cozy-fox-0defec.netlify.app

File: docs/console-api/experiments.mdx [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+title: Experiments
+sidebar_label: Experiments
+slug: /console-api/experiments
+description: Create feature experiments via HTTP requests.
+hide_table_of_contents: true
+custom_edit_url: null
+---
+import Rapidoc from "../../src/components/Rapidoc"
+
+<Rapidoc id="experiments" entity='experiments'/>
```
Comment (tore-statsig) [inline docs/console-api/experiments.mdx:11]: wow this is so cool. Now you can just specify the schema and it generates a doc page from that?  Gamechanger

**Review by daniel-statsig — APPROVED** [review]
nice

File: docs/console-api/models/index.js [hunk 2]
```diff
@@ -3,6 +3,8 @@ export default {
   autotune: require("./autotune.json"),
   dynamic_config: require("./dynamic_config.json"),
   dynamic_config_update: require("./dynamic_config_update.json"),
```
Comment (tore-statsig) [inline docs/console-api/models/index.js:5]: these were all using_underscores but now you did `camelCase` for `experimentMetric` - stick to one

**Review by tore-statsig — APPROVED** [review]
neato

File: docs/console-api/models/index.js (hunk 2 above)
Comment (jacob-statsig) [reply docs/console-api/models/index.js:5]: Will fix when I push the overrides docs, the user shouldn't be able to see this

File: docs/console-api/experiments.mdx (hunk 1 above)
Comment (jacob-statsig) [reply docs/console-api/experiments.mdx:11]: <3

**Pull request merged**